CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+## 1.5.0-beta.1
+
+## ✨ Features
+* GeolocateControl displays a `disabled` icon if user denies geolocation permission. [#8871](https://github.com/mapbox/mapbox-gl-js/pull/8871))
```

Comment:
nit: preferably entries all start with a verb that indicates what we did. so in this case, perhaps: "Add `disabled` icon to GeolocateControl if user denies geolocation permission."

Reply:
Also what makes this change in the "Features" section but the other change to GeolocateControl in the "Improvements" section? I'm having a hard time distinguishing between the two sections 🤔

Reply:
I moved it into the Features section, that does make more sense.